<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="C_perferential"
           tests="1"
           failures="1"
           errors="0"
           skipped="0"
           time="17.944">
  <properties />
            <testcase name="delBusiness" time="17.944" classname="com.wymall.test.testcases.preferential.business.BusinessPage_005_delBusiness_Test">
                <failure type="java.lang.AssertionError"
                  message="超时!! 10 秒之后还没找到元素 [By.xpath: //*[@id=&quot;couponTraderTable&quot;]//div[@class=&apos;el-table__fixed-right&apos;]//i[@class=&apos;el-icon-delete&apos;]]"
              >
<![CDATA[java.lang.AssertionError: 超时!! 10 秒之后还没找到元素 [By.xpath: //*[@id="couponTraderTable"]//div[@class='el-table__fixed-right']//i[@class='el-icon-delete']]
  org.testng.Assert.fail(Assert.java:94)
  com.wymall.test.utils.SeleniumUtil.waitForElementToLoad(SeleniumUtil.java:279)
  com.wymall.test.utils.SeleniumUtil.click(SeleniumUtil.java:135)
  com.wymall.test.utils.actions.impl.Click.doAction(Click.java:23)
  com.wymall.test.utils.SuperAction.parseExcel(SuperAction.java:277)
  com.wymall.test.testcases.preferential.business.BusinessPage_005_delBusiness_Test.delBusiness(BusinessPage_005_delBusiness_Test.java:8)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  java.lang.reflect.Method.invoke(Method.java:498)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
  org.testng.TestRunner.privateRun(TestRunner.java:767)
  org.testng.TestRunner.run(TestRunner.java:617)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:335)
  org.testng.SuiteRunner.access$000(SuiteRunner.java:37)
  org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:369)
  org.testng.internal.thread.ThreadUtil$2.call(ThreadUtil.java:64)
  java.util.concurrent.FutureTask.run(FutureTask.java:266)
  java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
  java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
  java.lang.Thread.run(Thread.java:748)
]]>
      </failure>

          </testcase>
  
  
  
</testsuite>
